runtime/src/bank.rs

```diff
@@ -188,21 +187,81 @@ impl HashAgeKind {
     }
 }
 
-#[derive(Default, Clone, PartialEq, Debug, Deserialize, Serialize)]
-struct UnusedAccounts {
-    unused1: HashSet<Pubkey>,
-    unused2: HashSet<Pubkey>,
-    unused3: HashMap<Pubkey, u64>,
+#[derive(Clone, Default)]
+pub(crate) struct BankFields {
+    pub(crate) blockhash_queue: BlockhashQueue,
+    pub(crate) ancestors: Ancestors,
+    pub(crate) hash: Hash,
+    pub(crate) parent_hash: Hash,
+    pub(crate) parent_slot: Slot,
+    pub(crate) hard_forks: HardForks,
+    pub(crate) transaction_count: u64,
+    pub(crate) tick_height: u64,
+    pub(crate) signature_count: u64,
+    pub(crate) capitalization: u64,
+    pub(crate) max_tick_height: u64,
+    pub(crate) hashes_per_tick: Option<u64>,
+    pub(crate) ticks_per_slot: u64,
+    pub(crate) ns_per_slot: u128,
+    pub(crate) genesis_creation_time: UnixTimestamp,
+    pub(crate) slots_per_year: f64,
+    pub(crate) unused: u64,
+    pub(crate) slot: Slot,
+    pub(crate) epoch: Epoch,
+    pub(crate) block_height: u64,
+    pub(crate) collector_id: Pubkey,
+    pub(crate) collector_fees: u64,
+    pub(crate) fee_calculator: FeeCalculator,
+    pub(crate) fee_rate_governor: FeeRateGovernor,
+    pub(crate) collected_rent: u64,
+    pub(crate) rent_collector: RentCollector,
+    pub(crate) epoch_schedule: EpochSchedule,
+    pub(crate) inflation: Inflation,
+    pub(crate) stakes: Stakes,
+    pub(crate) epoch_stakes: HashMap<Epoch, EpochStakes>,
+    pub(crate) is_delta: bool,
+}
+
+pub(crate) struct RefBankFields<'a> {
```

Comment:
Could you comment why `RefBankFields` is needed along side with just `BankFields`?

Reply:
Commented by my self https://github.com/solana-labs/solana/pull/10581#issuecomment-652905768